docs/dev/development.md

```diff
@@ -1,70 +1,90 @@
 
-The next step is to edit the `cbd-local/Profile` file with any editor and add the CB_SCHEMA_SCRIPTS_LOCATION environment variable which configures the location of SQL scripts that are in the 'core/src/main/resources/schema' directory in the cloned Cloudbreak git repository. Please note that the full path needs to be configured and env variables like $USER cannot be used.
+To start the complete Cloudbreak ecosystem on your machine just execute the following sequence of commands:
+```
+curl https://raw.githubusercontent.com/hortonworks/cloudbreak-deployer/master/install-dev | sh [-s branch] && cbd --version
+```
+Use the -s branch option for sh here in case you'd like to checkout another branch than master.
 
 ```
-export CB_SCHEMA_SCRIPTS_LOCATION=/Users/myusername/prj/cloudbreak/core/src/main/resources/schema
+docker-machine start default
+eval $(docker-machine env)
+
+cbd start
+cbd logs cloudbreak
 ```
+If everything went well then Cloudbreak will be available on http://192.168.99.100:3000. For more details and config parameters please check the documentation of [Cloudbreak Deployer](https://github.com/hortonworks/cloudbreak-deployer).
+
+The deployer has generated a `certs` directory under `cbd-local` directory which will be needed later on to set up IDEA properly.
 
-In order to kill Cloudbreak container running inside the boot2docker and redirect the Cloudbreak related traffic to the Cloudbreak running in IDEA use the followig command:
 
 ```
 cbd util local-dev
 ```
 
+In order to kill Cloudbreak container running in docker/boot2docker and redirect the Cloudbreak related traffic to the Cloudbreak running in IDEA use the followig command:
+
 ## IDEA
 
-Cloudbreak can be imported into IDEA as gradle project. Once it is done, you need to import the proper code formatter by using the __File -> Import Settings...__ menu and selecting the `idea_settings.jar` located in the `config` directory in Cloudbreak git repository.
+Configure -> Project Defaults -> Project Structure _> Set SDK to your Java version
+
+Cloudbreak can be imported into IDEA as gradle project. Once it is done, you need to import the proper code formatter by using the __File -> Import Settings...__ menu and selecting the `idea_settings.jar` located in the `config/idea` directory in Cloudbreak git repository.
 
 To launch the Cloudbreak application execute the `com.sequenceiq.cloudbreak.CloudbreakApplication` class with VM options:
 ```
 -XX:MaxPermSize=1024m
 -Dcb.cert.dir=FULL_PATH_OF_THE_CERTS_DIR_GENERATED_BY_CBD
 -Dcb.client.id=cloudbreak
--Dcb.client.secret=cbsecret2015
--Dcb.db.port.5432.tcp.addr=192.168.59.103
+-Dcb.client.secret=CB_SECRET_GENERATED_BY_CBD
+-Dcb.db.port.5432.tcp.addr=192.168.99.100
```

Comment:
This IP address is used for Virtualbox, xhyve allocates `192.168.64.3` for the first instance, but this parameter depends on the IP of the docker machine instance called `cbd`